### PR TITLE
fix minor errors on html tags causing errors

### PR DESCRIPTION
**Pull request by 274188A**
found I needed to make these changes in order for the project to compile so just giving you a heads up.  no need to adopt my pull exactly.

### CODE REVIEW

**Comment by lucabriguglia** [comment]
Thanks for your contribution @274188A, I'm not sure how the empty selected and value ended up there :-)
If you remove the changes from `src/Atlas.Client/Pages/Admin/Members/Edit.razor` I'll merge the pull request.

File: src/Atlas.Client/Pages/Admin/Members/Edit.razor [hunk 1]
```diff
@@ -37,47 +37,47 @@ else
                 <div class="col-md-6">
                     <div class="form-group">
                         <label class="control-label font-weight-bold">@Loc["Status"]:</label>
-                        <label class="">@Model.Info.Status</label>
```
Comment (lucabriguglia) [inline src/Atlas.Client/Pages/Admin/Members/Edit.razor:40]: Why is this class causing issues? 
And anyway you left the property empty.
Please remove this changes and I'll merge the pull request.

**Review by lucabriguglia — CHANGES_REQUESTED** [review]

**Comment by lucabriguglia** [comment]
Why is that class a problem? What is your issue?

File: src/Atlas.Client/Pages/Admin/Members/Edit.razor (hunk 1 above)
Comment (274188A) [reply src/Atlas.Client/Pages/Admin/Members/Edit.razor]: having `class=""`  cause a compile error for me

Comment (274188A) [reply src/Atlas.Client/Pages/Admin/Members/Edit.razor]: for example this:

`<label class="">@Model.Info.Status</label>`

cause this:

image

**Comment by 274188A** [comment]
happy for you to chuck my changes and fix your end if easier...

**Comment by lucabriguglia** [comment]
Sorry @274188A, I completely messed up the commit. You are absolutely right, please re-submit the pull request with your changes.

**Comment by lucabriguglia** [comment]
I have committed some changes in the meantime including removing the empty selected and option value.

**Comment by 274188A** [comment]
starting a fresh pull

**Comment by 274188A** [comment]
you latest did the trick - thanks - no need for another pull

**Pull request closed without merging**